filament/backend/src/opengl/OpenGLProgram.h

```diff
@@ -78,6 +85,13 @@ class OpenGLProgram : public HwProgram {
         GLuint program = 0;
     } gl;                                               // 4 bytes
 
+    utils::FixedCapacityVector<PushConstantBundle> const* getPushConstants() {
```

Comment:
that should be getPushConstant(stage). I think let's bite the bullet and implement the stages here.

Reply:
done in OpenGLDriver.  I encapsulated the offset+array in a new struct `PushConstantBundle`, which can be stashed in OpenGLDriver when pipeline is bound, and queried for constant when `setPushConstant` is called.